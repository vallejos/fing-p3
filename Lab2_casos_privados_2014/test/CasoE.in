24
1 -1
2 -1
3 -1
4 -1
5 -1
0 -1
7 14 -1
8 -1
6 -1
10 -1
11 -1
12 -1
13 -1
9 14 -1
15 -1
16 -1
17 -1
18 -1
14 -1
20 -1
21 -1
22 -1
23 -1
19 -1
LaVela 100 Hereford 100 ChalaMadre 500 -1 -1
LaVela 100 Hereford 1 ChalaMadre 500 -1 -1
LaVela 100 Hereford 100 ChalaMadre 500 -1 -1
LaVela 100 Hereford 1 -1 -1
LaVela 100 Hereford 1 -1 -1
LaVela 100 Hereford 1 -1 -1
Trotsky 50 OnceTiros 10 LaTrampa 1000 -1 -1
Trotsky 50 OnceTiros 10 -1 -1
Trotsky 50 OnceTiros 45 -1 -1
Estomagos 130 Buitres 10 LaAbuelaCoca 300 -1 -1
Estomagos 100 Buitres 100 OnceTiros 1000 -1 -1
Estomagos 130 Buitres 1 OnceTiros 1000 -1 -1
Estomagos 100 Buitres 4 OnceTiros 1000 -1 -1
Estomagos 130 Buitres 50 OnceTiros 1000 -1 -1
Buitres 90 ElCuarteto 10 LaAbuelaCoca 1000 -1 -1
Buitres 60 ElCuarteto 110 LaAbuelaCoca 1000 -1 -1
Buitres 90 ElCuarteto 1 LaAbuelaCoca 300 -1 -1
Buitres 100 ElCuarteto 5 -1 -1
Buitres 100 ElCuarteto 2 LaAbuelaCoca 1200 -1 -1
NTVG 300 Hereford 10 ElCuarteto 10 -1 -1
NTVG 20 Hereford 100 ElCuarteto 1000 -1 -1
NTVG 100 Hereford 1 -1 -1
NTVG 200 Hereford 3 ElCuarteto 2410 -1 -1
NTVG 150 Hereford 5 ElCuarteto 5 -1 -1
identificarColecciones 10 LaVela Trotsky Estomagos Buitres NTVG ChalaMadre OnceTiros LaAbuelaCoca LaTrampa ElCuarteto
identificarColecciones 1 LaAbuelaCoca
salir
