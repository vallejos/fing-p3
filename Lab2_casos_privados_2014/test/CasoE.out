Tarea 2 - 2014 - Programacion 3.

>0 -> LaVela
1 -> Trotsky
2 -> Estomagos
3 -> Buitres
4 -> NTVG
>0 -> 
1 -> 
2 -> 
3 -> 
4 -> 
>FIN.
